YSE_App/frb_init.py

```diff
@@ -0,0 +1,28 @@
+""" Methods to init the DB """
+
+from YSE_App.models import TransientStatus
+from YSE_App.data_utils import add_or_grab_obj
+
+def init_status(user):
+
+    # Delete all existing
+    TransientStatus.objects.all().delete()
+
+    # Create the FRB ones
+    all_status = [\
+        'Unassigned', # Likely just ingested although most will be given PublicPATH
+        'PublicPATH', # Needs to be run through PATH with public data
+        'Image', # Needs deeper imaging
+        'Spectrum', # Needs spectroscopy for redshift
```

Comment:
I guess there's needing spectroscopy for redshift and needing a mask for foreground analyses. Do we want to differentiate?

Reply:
FFFF-PZ will not monitor fore-ground follow-up